json-parser/src/test/java/org/uniprot/core/json/parser/uniparc/UniParcCrossRefJsonConfigTest.java

```diff
@@ -3,6 +3,7 @@
 import static org.junit.jupiter.api.Assertions.fail;
 import static org.uniprot.core.json.parser.uniparc.UniParcEntryTest.getCompleteUniParcEntry;
 
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
```

Comment:
Can you please remove this import (I do not think this is being used)

Reply:
This is being used by Java time module.